src/proto/Sui.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+syntax = "proto3";
+
+package TW.Sui.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message SigningInput {
+    bytes private_key = 1;
+    // Obtain by calling any write RpcJson on SUI
+    string unsigned_tx = 2;
```

Comment:
this `unsigned_tx` is barely unreadable, at least add simple messages like `transfer`, `contract call`